runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -52,17 +52,9 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 */
 private:
 	void AssertBadElementSize();
+	void AssertArrayletIsDiscontiguous(J9IndexableObject *objPtr);
 #if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
 	void AssertNotEmptyArrayletLeaves(UDATA sizeInElements, UDATA arrayletLeafCount);
-	MMINLINE bool
-	isOneArrayletLeafWithNULL(J9IndexableObject *spine, UDATA arrayletLeafCount, UDATA sizeInElements)
-	{
-		UDATA arrayletLeafSize = _omrVM->_arrayletLeafSize;
-		UDATA dataSize = getDataSizeInBytes(spine);
-		AssertNotEmptyArrayletLeaves(sizeInElements, arrayletLeafCount);
```

Comment:
I believe this was the only user of AssertNotEmptyArrayletLeaves(), so that it can be removed.